templates/vue/src/components/tyde/activities/CircleOfSupport/CommunityView/onboarding/ObFinishView.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+  <cos-modal show>
+    <b-container class="modal-cont">
+      <b-row align-h="center">
+        <b-col
+          v-for="connection in threeConnections()"
+          :key="connection.id"
+          align-self="center"
+          class="ob-connection-box"
+        >
+          <p class="ob-connection">{{ connection.name }}</p>
+          <h1>{{ connection.avatar }}</h1>
+        </b-col>
+      </b-row>
+      <b-row align-h="center">
+        <h3>
+          Great job! Here are some of
+          <br />
+          your connections.
+        </h3>
+      </b-row>
+      <b-row align-h="center">
+        <h5 class="ob-secondary">
+          You'll be able to edit and add more at any time.
+        </h5>
+      </b-row>
+      <b-row align-h="center">
+        <b-col v-show="visibleConnections.length >= 4" class="ob-connection-box">
+          <p class="ob-connection">
+            {{ (visibleConnections[3] && visibleConnections[3].name) || "" }}
```

Comment:
This is a very "reacty" way of doing things :) It'd be better to use v-if or v-show on the element instead.